pyproject.toml

```diff
@@ -1,8 +1,43 @@
-[tool.poetry]
-name = "specfem2d_kokkos"
+[build-system]
+requires = ["scikit-build-core>=0.10", "pybind11"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "SPECFEMPP"
 version = "0.1.0"
 description = "Kokkos implementation of SpecFEM2D code"
-authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+authors = [
+  { name = "Rohit R Kakodkar", email = "rohit_kakodkar@princeton.edu" },
+  { name = "Lucas Sawade", email = "lsawade@princeton.edu" },
+  { name = "Congyue Cui", email = "ccui@princeton.edu" }
+]
+requires-python = ">=3.9"
+# classifiers = [
```

Comment:
We can remove these comments

Reply:
The comments will be removed in the next pull request